features/partners.feature

```diff
@@ -5,11 +5,10 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
   Dit bevat de gegevens over de relatie (huwelijk of partnerschap) plus enkele identificerende eigenschappen van de persoon waarmee het huwelijk of partnerschap is aangegaan.
 
   # RvIG levert personen in de vorm GbaPersoon, waarin alleen onbewerkte GBA-V gegevens worden geleverd
-  # Actuele partners is (huwelijk / partnerschap is niet ontbonden) worden geleverd
-  # Als er een (of meer) actuele pertner is worden geen ontbonden huwelijken/partnerschappen geleverd
-  # Als er alleen ontbonden huwelijken/partnerschappen zijn levert de RvIG alleen de ex-partner met de meest recente datum ontbinding
+  # Dit betekent dat in 'partners' een actuele partner of een ontbonden partner kan bevatten
   # Een proxy vertaalt dit naar de vorm Persoon, waarin sommige gegevens in bewerkte vorm worden opgenomen en waarin informatievragen kunnen zitten
   # Een voorbeeld van een informatievraag is naam.aanschrijfwijze, waarin op basis van de aanduiding naamgebruik, de naam van de persoon en de naam van de (ex)partner een aanschrijfnaam wordt samengesteld
+  # De proxy verwijdert de partners met datum ontbinding die geleverd zijn in de GbaPersoon
 
   @gba
   Rule: Het gegeven 'partners' in GbaPersoon kan een ontbonden huwelijk of partnerschap bevatten
```

Comment:
ik ben niet zo gelukkig met deze rule. Wat is nu precies de regel die hier gecommuniceerd wordt? Dat het mogelijk is dat partners een ontbonden partner bevat? Maar wanneer dan wel of niet?

De scenario's tonen het volgende:
- Als er een (of meer) actuele partner is/zijn worden deze geleverd en worden geen ontbonden huwelijken/partnerschappen geleverd
- Als er alleen ontbonden huwelijken/partnerschappen zijn levert de RvIG alleen de ex-partner met de meest recente datum ontbinding
- Voor een ontbonden huwelijk of partnerschap worden de aangaanHuwelijkPartnerschap gegevens gehaald uit de meest recente historische categorie 55

Reply:
Ik heb de rule anders geformuleerd (alleen voor de levering van actuele partners) en een rule toegevoegd voor het leveren van ontbonden partnerschappen.

Reply:
Ik heb de rule andersgeformuleerd zodat hij allen voor de actele huwelij/partnerschappen geldt. Ik heb ook een Rule toegevoegd voor ontbonden huwelijken/partnerschappen